acceptance/launcher_test.go

```diff
@@ -353,3 +363,14 @@ func assertOutput(t *testing.T, cmd *exec.Cmd, expected ...string) {
 		}
 	}
 }
+
+func copyPosixFixture(t *testing.T) {
+	windowsContext := filepath.Join("testdata", "launcher", "windows")
+	h.AssertNil(t, os.MkdirAll(filepath.Join(windowsContext, "exec.d"), 0755))
+	h.RecursiveCopy(
+		t,
+		filepath.Join("testdata", "launcher", "posix", "exec.d"),
+		filepath.Join(windowsContext, "exec.d"),
+	)
+	os.RemoveAll(filepath.Join(windowsContext, "container", "layers", "0.5_buildpack", "some_layer", "exec.d", "exec.d-checker", ".gitkeep"))
```

Comment:
I'm not totally sure that it's necessary, but I saw it here: https://github.com/buildpacks/lifecycle/blob/7f1c2db042cfef1a5adbe052a89d30a15a63aa80/acceptance/testdata/launcher/posix/Dockerfile#L11 and when trying to replicate with `del /F` in the Windows Dockerfile I got a permissions issue. I'll experiment to see if it's required...

Reply:
It is necessary... but I couldn't delete it inside the docker build. Trying to create the directory in the build instead...

Reply:
Because I couldn't run `del /F <path to .gitkeep>` during the build I moved the .gitkeep up two directories and created empty directories in the Dockerfile: https://github.com/buildpacks/lifecycle/pull/534/files#diff-b45234da4232089107bcd78584060e81019e23a33c8fe5418e81b2c3cefacf69R8. IDK if there's a better way to do it...